common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -61,6 +68,23 @@ public static void logMessage(Level level, MessageAuditLogEntry.Builder entryBui
             .setComponent(buildProperties.getArtifact())
             .setVersion(buildProperties.getVersion())
             .build());
+    Map<String, Object> fluentDLogs = new HashMap<>();
+
+    if (logForwarderProperties.isEnabled()) {
+      FLUENT_LOGGER =
+          FluentLogger.getLogger(
```

Comment:
`To create Fluent Logger instances, users need to invoke getLogger method in FluentLogger class like org.slf4j, org.log4j logging libraries. The method should be called only once`

Does this mean you are creating a new logger every time? Why did you create a static instance?

Reply:
Updated, creating through constructor now.